src/domScripts/startGameButton.js

```diff
@@ -0,0 +1,8 @@
+const startGameButton = document.querySelector('newGameButton');
```

Comment:
mały błąd ale możliwe, że przez to Ci nie działa. QuerySelector szuka pierwszego elementu według selektora jak z CSS. tutaj wpisałeś "newGameButton", więc szuka elementu o nazwie "newGameButton". Żeby szukać po klasie musisz dodać kropeczkę przed więc spróbuj zamienić jak niżej 
```suggestion
const startGameButton = document.querySelector('.newGameButton');
```